lifecycle-operator/controllers/lifecycle/keptnevaluation/controller.go

```diff
@@ -70,14 +70,14 @@ type KeptnEvaluationReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.12.2/pkg/reconcile
 func (r *KeptnEvaluationReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-
-	r.Log.Info("Reconciling KeptnEvaluation")
+	controllerInfo := controllercommon.GetControllerInfo(req)
+	r.Log.Info("Reconciling KeptnEvaluation", controllerInfo)
```

Comment:
passing an odd number of additional arguments to the log.Info function will cause it to crash. I've seen you already pass the info object correctly in other places, so let's double check this is the case everywhere we make use of this :) 
```suggestion
	r.Log.Info("Reconciling KeptnEvaluation", "controllerInfo", controllerInfo)
```